core/src/consensus.rs

```diff
@@ -631,7 +631,6 @@ impl Tower {
         }
     }
 
-    #[cfg(test)]
```

Comment:
think you can use the new guy here:
```
#[cfg(feature = "dev-context-only-utils")]
```

Reply:
didn't even realize this existed, works

Reply:
huh works locally but is failing CI: https://buildkite.com/anza/agave/builds/1225#018e68a5-2942-41ba-973a-bf89c03c8950

Reply:
huh strange. I'm fine if you push it without the flag for now

Reply:
cool, removed